README.md

```diff
@@ -1 +1,237 @@
-# Vocdoni SDK
\ No newline at end of file
+# Vocdoni SDK
+
+The Vocdoni SDK is the easiest way to use [our new API][Vocdoni API], allowing
+you to easily create new voting processes, fetch their information and vote them.
+
+## Prerequisites
+
+Obviously you'll need a working nodejs environment, but other than that, you're
+free to use any package manager (either npm, yarn, pnpm...). Let's start by
+adding the SDK to your project:
+
+~~~bash
+# with yarn
+yarn add vocdoni-sdk
+# with npm
+npm i vocdoni-sdk
+# with pnpm
+pnpm add vocdoni-sdk
+~~~
+
+In case you want to create processes or vote to them, you'll need a signer to
+sign transactions. You should be able to use any kind of [ethers] signer.
+
+For this readme examples, the signer bootstraping will be ignored and you'll
+just see a `signer` constant.
+
+## SDK Usage
+
+The entry point is the SDK constructor, it will instantiate a new client
+connected to the specified API endpoint:
+
+~~~ts
+//                                   api endpoint URL                 signer
+const client = new VocdoniSDKClient('https://api-dev.vocdoni.net/v2', signer)
+~~~
+
+### Registering account
+
+Before creating any new processes, you should register your account against
+our blockchain (vochain):
+
+~~~ts
+(async () => {
+  const info = await client.createAccount()
+  console.log(info) // will show account information
+})();
+~~~
+
+The `createAccount` method will try to fetch an existing account first and, if
```

Comment:
Maybe it would be better to explain that the registered account will be the one used for the `signer` when creating the client, because in the call there is no explicit account/address involved

Reply:
True. I'll improve this part.